department-of-veterans-affairs/caseflow
# Mkerns/caseflow 868 last updated poa

Resolves [CASEFLOW-868](https://vajira.max.gov/browse/CASEFLOW-868)

### Description
Added PoaRefresh component to show user the most recent last synced at date for POA data.

### Acceptance Criteria
- [x] Given a user advances to the Case Details page
  - [x] When they view the POA section 
  - [x] Then the last updated date displays 
- [x] Work for this should be placed behind the poa_sync_date Feature Toggle

### Testing Plan
1. Enable the poa sync date feature toggle `FeatureToggle.enable!(:poa_sync_date) `
2. Go to Case Details page
2. Scroll to Power Of Attorney Details Section
3. Verify that the last sync date and text is visable

### User Facing Changes

 BEFORE|AFTER
 
![image](https://user-images.githubusercontent.com/69862429/115259901-4a169d00-a100-11eb-9a56-8d14bdd7a33d.png)
|
![image](https://user-images.githubusercontent.com/69862429/115260503-ce692000-a100-11eb-876a-9534a70b8e06.png)


### Storybook Story
*For Frontend (Presentational) Components*
